scanners/sslyze.py

```diff
@@ -602,6 +699,9 @@ def queue(command):
     if options.get("sslyze-certs", True) is True:
         queue(CertificateInfoScanCommand())
 
+    if options.get("sslyze-reneg", True) is True:
```

Comment:
This should just be `if options.get("sslyze-reneg", True):`

Reply:
:eagle: :eye: